pages/k8s/cni-calico.md

```diff
@@ -64,7 +67,8 @@ that would apply to `charmed-kubernetes` to this bundle also.
 |---------------------|--------|------------------------------------------|----------------------------------------------------------------|
 | calico-node-image   | string | docker.io/calico/node:v3.6.1             | The image id to use for calico/node                            |
 | calico-policy-image | string | docker.io/calico/kube-controllers:v3.6.1 | The image id to use for calico/kube-controllers                |
-| ipip                | string | Never                                    | IPIP mode. Must be one of "Always", "CrossSubnet", or "Never". |
+| ipip                | string | Never                                    | IPIP encapsulation mode. Must be one of "Always", "CrossSubnet", or "Never". This is incompatible with VXLAN encapsulation. If VXLAN encapsulation is enabled, then this must be set to "Never". |
```

Comment:
Perhaps there is some reason for grouping certain options together in this table, but as it gets longer, I feel it would benefit from being sorted alphabetically to make things easier to find. Any objection?

Reply:
Ah yeah, that makes sense. No objections here.